jbmc/src/java_bytecode/java_bytecode_convert_class.cpp

```diff
@@ -118,11 +118,43 @@ class java_bytecode_convert_classt:public messaget
   // see definition below for more info
   static void add_array_types(symbol_tablet &symbol_table);
 
+  /// Check if a method is an overlay method by searching for
+  /// `ID_overlay_method` in its list of annotations.
+  ///
+  /// An overlay method is a method with the annotation
+  /// \@OverlayMethodImplementation. They should only appear in
```

Comment:
:+1: Thanks for explaining this, I wasn't sure if `@OverlayClassImplementation` automatically turned all methods into overlay methods, of if within an overlay class methods could still be declared to be overlay methods or not.